cluster/sync.sh

```diff
@@ -2,80 +2,101 @@
 
 set -ex
 
+kubectl=./cluster/kubectl.sh
+manifests_dir=deploy
+
 function getDesiredNumberScheduled {
-    echo $(./cluster/kubectl.sh get daemonset -n nmstate $1 -o=jsonpath='{.status.desiredNumberScheduled}')
+    $kubectl get daemonset -n nmstate $1 -o=jsonpath='{.status.desiredNumberScheduled}'
 }
 
 function getNumberAvailable {
-    numberAvailable=$(./cluster/kubectl.sh get daemonset -n nmstate $1 -o=jsonpath='{.status.numberAvailable}')
+    numberAvailable=$($kubectl get daemonset -n nmstate $1 -o=jsonpath='{.status.numberAvailable}')
     echo ${numberAvailable:-0}
 }
 
-function consistently {
+function eventually {
+    timeout=30
+    interval=5
     cmd=$@
-    retries=3
+    echo "Checking eventually $cmd"
+    while ! $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 1
+        fi
+    done
+}
+
+function consistently {
+    timeout=10
     interval=1
-    cnt=1
-    while [[ $cnt -le $retries ]]; do
-        $cmd
+    cmd=$@
+    echo "Checking consistently $cmd"
+    while $cmd; do
         sleep $interval
-        cnt=$(($cnt + 1))
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 0
+        fi
     done
 }
 
 function isOk {
     desiredNumberScheduled=$(getDesiredNumberScheduled $1)
     numberAvailable=$(getNumberAvailable $1)
+    [ "$desiredNumberScheduled" == "$numberAvailable" ]
+}
 
-    if [ "$desiredNumberScheduled" == "$numberAvailable" ]; then
-        echo "$1 DS is ready"
-        return 0
+deploy() {
```

Comment:
Please follow the style of function definitions we use in the rest of the script